README.md

```diff
@@ -74,6 +74,7 @@ A curated list of amazingly awesome open source resources for broadcasters inspi
 
 ## Multimedia content processing
 
+* [codem-isoboxer] (https://github.com/madebyhiro/codem-isoboxer) A small browser-based MPEG-4 (ISOBMFF) parser.
```

Comment:
I have made the change, but note that the original project also has not capitals and neither do :
Companion Screens

  *   dvbcss-synctiming<https://github.com/BBC/dvbcss-synctiming> - dvbcss-synctiming is a system for measuring how accurately a TV or companion are synchronised.
  *   pydvbcss<https://github.com/BBC/pydvbcss> - pydvbcss is library implementing the DVB Companion Screens and Streams protocols for accurately synchronising media playback between TVs and companions.
--
On the category: I thought it was closest to the UPipe case, which also is primarily for players (but not only). Happy to see it moved around as you think best, though!


From: Michael Barroco [mailto:notifications@github.com]
Sent: jeudi 9 juillet 2015 11:44
To: ebu/awesome-broadcasting
Cc: De Jong, Frans
Subject: Re: [awesome-broadcasting] add 3 projects (#8)


In README.md<https://github.com/ebu/awesome-broadcasting/pull/8#discussion_r34239040>:

> @@ -74,6 +74,7 @@ A curated list of amazingly awesome open source resources for broadcasters inspi

>

>  ## Multimedia content processing

>

> +* [codem-isoboxer] (https://github.com/madebyhiro/codem-isoboxer) A small browser-based MPEG-4 (ISOBMFF) parser.

  1.  Could you please capitalize the first letter of the project.
  2.  Not sure about the category, since it is a browser package, shouldn't it go to the Media Player section?

—
Reply to this email directly or view it on GitHub<https://github.com/ebu/awesome-broadcasting/pull/8/files#r34239040>.
------------------------------------------------------------------------------

**************************************************
This email and any files transmitted with it are confidential and intended solely for the use of the individual or entity to whom they are addressed.
If you have received this email in error, please notify the system manager. This footnote also confirms that this email message has been swept by the mailgateway
**************************************************